specification/sql/resource-manager/Microsoft.Sql/preview/2022-08-01-preview/ManagedLedgerDigestUploads.json

```diff
@@ -0,0 +1,335 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-08-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/managedInstances/{managedInstanceName}/databases/{databaseName}/ledgerDigestUploads": {
+      "get": {
+        "tags": [
+          "ManagedLedgerDigestUploads"
+        ],
+        "description": "Gets all ledger digest upload settings on a database.",
+        "operationId": "ManagedLedgerDigestUploads_ListByDatabase",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
```

Comment:
>v1 [](http://example.com/codeflow?start=37&length=2)

Recommended to use v3 or v4 common-types now in all of your new changes, they have recent updated model for the standard types.

Reply:
@visingla-ms for the common types v4, we have had the exception from ARM side for this version. 
https://github.com/Azure/azure-rest-api-specs/pull/21415#discussion_r1026792600